docs/SceneWalkthrough/PlayerReportsAndSanctionsWalkthrough.md

```diff
@@ -0,0 +1,13 @@
+## **Player Reports and Sanctions Demo**
+This demo showcases the reports interface, and sanctions interface. This is done through a report menu, and a display of any current sanctions on the user's account
```

Comment:
I think the comma after "interface" is unneeded. @RedMazer probably knows the answer.